### PR TITLE
adds developer update blog for June

### PR DESCRIPTION
**Pull request by SvenMuller95**

### CODE REVIEW

File: blog/2024-07-01-envio-community-update-june-2024.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+## Upcoming Events 🗓️
+
+- [Encode Club - Brussels Hacker House](https://www.encode.club/brussels2024): 8th - 10th July, 2024
+- [ETHCC Brussels](https://ethcc.io/): 8th - 11th July, 2024
+- [Modular Summit 3.0](https://modularsummit.dev/): 11th - 13th July, 2024
+
+## Developer Tutorials🧑‍💻
+
+- How to index ERC-20 transfers on Celo up to 1000x faster than RPC using Envio⚡
+- Indexing on Fuel: No-code Contract Import Sway Farm Tutorial
```
Comment (moose-code) [inline blog/2024-07-01-envio-community-update-june-2024.md:62]: Do we want to add hyperlinks probably to these exact tutorials right?

File: blog/2024-07-01-envio-community-update-june-2024.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+Excited to announce that Peppy Finance has integrated Envio’s data indexing capabilities to power its decentralized perpetual future swap exchange on Shimmer, delivering a seamless UX to its end users.
+
+Learn more about the integration in our case study [blog](https://docs.envio.dev/blog/envio-real-time-indexing-powers-peppy-finance).👇
+
+## New HyperSync Network Support ⚡
+
+We’re excited to announce that Envio HyperSync has expanded enhanced indexing support for developers building on Photon, Rootstock and Darwinia including their canary network Crab.
+
+To see the full list of currently supported chains on HyperSync visit our docs.
```
Comment (moose-code) [inline blog/2024-07-01-envio-community-update-june-2024.md:42]: Hyperlink to appropriate place?

**Review by moose-code — APPROVED** [review]
Great work!! Just some hyperlinks to add :)

**Comment by Jordy-Baby** [comment]
Hey @moose-code I've updated to include Hyperlinks! Thank ser! 🙏

**Pull request merged**